TimeSeriesAnalysis.m

```diff
@@ -53,16 +54,26 @@
 %%
 %% 4: Only calculate for given GNSS station
 %%
-exclusionOption = 1;
+exclusionOption = 3;
 
-%% (When exclusionOption=4, only calculate for this GNSS/5D station)
-exclusiveGPSNR = 'KOKI';
+%% (When exclusionOption=4, only calculate for this GNSS station)
+exclusiveGPSNR = 'FALT';
 
 %%############################################################################
 %% MINIMUM NUMBER OF OBSERVATIONS:
 %% Set the minimum number of observation for station to get included in analysis.
-%% min_points=2 requires three observations per station.
-min_points = 1; 
+%% min_points=2 requires three observations per station. 
+min_points = 2; 
+##prompt = 'Enter minimum amount of points. Type 2 to require min. three obs. per station (default)';
+##dlgtitle = 'Input';
+%USER PROMPT MIN POINTS:

```

Comment:
Har du lavet det minimale antallet af punkter til analyse til en user input prompting?

Reply:
Det nævnte jeg også kort til mødet, at man enten kunne have min. antal punkter som en user prompt eller bare hardcode den til at kræve min. 3 punkter. Lige nu er den hardcodet til min. 3 punkter, men jeg har ladet den udkommenterede kode for user prompt stå hvis nu det bliver nødvendigt/rart at have senere. Skal jeg bare fjerne det?

Den fulde kode er:
% USE FOLLOWING CODE FOR PROMPTING USER OF min_points:
##prompt = 'Enter minimum amount of points. Type 2 to require min. three obs. per station (default)';
##dlgtitle = 'Input';
%USER PROMPT MIN POINTS:
%min_points = str2double(inputdlg(prompt,dlgtitle)); %str2double converts from cell to double.

%NOTE! If min_points = 1, you will get alot of division with zeros in ligreg, lingregplot and TimeSeriesAnalysis (around line 414), 
%as often the degrees of freedom will be equal to 0. (df = N-2), so if N=2, df = 0.